scipy/spatial/distance.py

```diff
@@ -2294,7 +2294,8 @@ def pdist(X, metric='euclidean', *, out=None, **kwargs):
 
     X = _asarray(X)
     if X.ndim != 2:
-        raise ValueError(f'A 2-dimensional array must be passed. (Shape was {X.shape}).')
+        raise ValueError('A 2-dimensional array must be passed. '
+                         f'(Shape was {X.shape}).')
```

Comment:
It'd be nice to drop the changes to this file, since it's style-only in a different module. Can be done in a separate PR if this bothers your local setup somehow.

Reply:
Yeah it came up with `spin lint`. I'll take it out.